_episodes_rmd/03-basics-factors-dataframes.Rmd

```diff
@@ -638,7 +641,7 @@ In this exercise, we will leave the title of the data frame as
 **Ecoli_metadata**, and there are no other options we need to adjust. Click the
 <KBD>Import</KBD> button to import the data.
 
-Finally, let's check the first few lines of the `Ecoli_metadata` metadata data
+Finally, let's check the first few lines of the `Ecoli_metadata` data
```

Comment:
I am not sure I think it's a good idea to use readxl here -- why not just export it as a csv and use that, with `read.csv()`? if this is happening on a server remotely, there's no way to view an xlsx file, and I think it encourages a bad practice (not using plain text whenever possible).

Reply:
This is a good point and I agree. I would actually prefer not covering spreadsheet import at all and sticking with csv, but I think this is @JasonJWilliamsNY's content, so I want to get his input before making a large change here.